cmd/staking/root.go

```diff
@@ -0,0 +1,173 @@
+package main
+
+import (
+	"bytes"
+	"encoding/json"
+	"fmt"
+	"io/ioutil"
+	"math/big"
+	"net/http"
+	"os"
+	"path"
+	"strconv"
+
+	"github.com/ethereum/go-ethereum/common/hexutil"
+	"github.com/ethereum/go-ethereum/rlp"
+	"github.com/harmony-one/bls/ffi/go/bls"
+	"github.com/harmony-one/harmony/accounts"
+	"github.com/harmony-one/harmony/accounts/keystore"
+	"github.com/harmony-one/harmony/internal/common"
+	"github.com/harmony-one/harmony/shard"
+	staking "github.com/harmony-one/harmony/staking/types"
+	"github.com/spf13/cobra"
+)
+
+const (
+	keystoreDir = ".hmy/keystore"
+	stakingRPC  = "hmy_sendRawStakingTransaction"
+)
+
+type staker struct{}
+
+var (
+	queryID       = 0
+	s             = &staker{}
+	localNetChain = big.NewInt(2)
+	dAddr         = common.ParseAddr(testAccount)
+)
+
+const (
+	// Harmony protocol assume beacon chain shard is only place to send
+	// staking, later need to consider logic when beacon chain shard rotates
+	stakingShard        = 0
```

Comment:
move it to upper level, so we can share this variable

Reply:
This is a temp CLI that later will remove once staking work done